src/main/java/racingcar/view/output/OutputView.java

```diff
@@ -0,0 +1,48 @@
+package racingcar.view.output;
+
+import java.util.List;
+import java.util.Map;
+import racingcar.model.Car;
+import racingcar.view.output.dto.RacingResponse;
+
+/**
+ * @author : jiffyin7@gmail.com
+ * @since : 24. 10. 27.
+ */
+public class OutputView {
+  private static final String EXECUTE_RESULT = "실행 결과";
+  private static final String WINNER_PREFIX = "최종 우승자 : ";
+  private static final String WINNER_DELIMITER = ", ";
+  private static final String SEPARATOR = " : ";
+  private static final String POSITION_COUNTER = "-";
+
+  public static void printExecuteResult(){
+    System.out.println(EXECUTE_RESULT);
+  }
+
+  private static void printCarStatus(Car car) {
+    StringBuilder stringBuilder = new StringBuilder(
+        car.getName())
+        .append(SEPARATOR)
+        .append(POSITION_COUNTER.repeat(car.getPosition()));
+    System.out.println(stringBuilder);
+  }
+
+  public static void printRoundResult(List<Car> cars){
+    for (Car car : cars){
+      printCarStatus(car);
+    }
+    System.out.println();
```

Comment:
```suggestion
cars.forEach(System.out::println);
        System.out.println();
```
이렇게하면 조금더 깔끔하게 사용할 수 있습니다

Reply:
오 람다식 하나 배워가네요.